tardis/model/base.py

```diff
@@ -0,0 +1,14 @@
+class Radial1DModel(object):
```

Comment:
@wkerzendorf What's the plan for `config.structure.volumes`? Where should that live? To me it feels right to put it into the model, too.